File: BoxOfCodes/UriOnlineJudge/1002.py
import math

area = None
Raio = None

def read_numeric():
  try:
    # read for Python 2.x
    return float(raw_input())
  except NameError:
    # read for Python 3.x
    return float(input())


Raio = read_numeric()

area = str("{:0.4f}".format((3.14159 * (math.pow(Raio, 2)))))

print ("A=" + area)

File: BoxOfCodes/UriOnlineJudge/1002.cs
using System;

class URI
{

    static void Main(string[] args)
    {
        double raio, area;
        double pi = 3.14159;

        raio = double.Parse(Console.ReadLine());

        area = (raio * raio) * pi;
        Console.WriteLine($"A={area:F4}");

    }

}